code/modules/clothing/head/helmet.dm

```diff
@@ -80,6 +77,12 @@
 	can_toggle = 1
 	toggle_cooldown = 20
 	activation_sound = 'sound/items/WEEOO1.ogg'
+	
+/obj/item/clothing/head/helmet/justice/attack_self()

```

Comment:
the real fix would be to put:

```
while(up)
    playsound(junk,junk,junk,junk)
    sleep(junk)
```
Under an ```if(activation_sound)```
since activation_sound is not a helmet/justice level variable, you've essentially removed that functionality from every other hat.